test/setup/server_setup.go

```diff
@@ -1476,6 +1496,13 @@ func (s *Server) URL() string {
 	return "http://" + s.address + ":" + strconv.FormatInt(int64(s.nodePort), 10)
 }
 
+func (s *Server) PrometheusURL() string {
+	s.mu.RLock()
+	defer s.mu.RUnlock()
+
+	return "http://" + s.address + ":" + strconv.FormatInt(int64(s.prometheusPort), 10)
+}
+
```

Comment:
do we have a test with TLS? the scheme should be http/https depending on whether security is on

Reply:
I've added such test